sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1268,3 +1461,140 @@ def display_data(self):
                                    tableSpec)
       res['table'] = DisplayDataItem(tableSpec, label='Table')
     return res
+
+
+@experimental()
+class PassThroughThenCleanup(PTransform):
```

Comment:
Does this class need to be public? If it should be public, it should be in a different file. If not, let's make it private and only use it in this file.

Reply:
I don't think it needs to be public. I'll make it private then.